client/app/(dashboard)/weekly-summary/page.tsx

```diff
@@ -47,10 +47,13 @@ function RecentSummaries() {
 				</CardHeader>
 				<CardContent>
 					<div className="space-y-3">
-						{Array.from({ length: 3 }).map((_, i) => (
+						{Array.from(
+							{ length: 3 },
+							(_, i) => `recent-summary-skeleton-${i + 1}`,
```

Comment:
ah nice, did not think about the +1 hack of getting this from "definitvely const" to "likely const".
(at least I think this is why this might just work)

Reply:
it's dumb as fuck, but wtf biome...